cclib/method/ddec.py

```diff
@@ -466,22 +520,22 @@ def condition_densities(self):
                     upperbigPhi = numpy.PINF
                     upperphi = numpy.PINF
 
-            iter = 0
-            while fitphi and iter < 50:
+            iteration = 0
+            while not phiconverged and iteration < 50:
```

Comment:
Do you think a for loop and break on convergence would be more natural here?

Reply:
Yes. That would remove the need of `phiconverged`. Thanks.